modules/sdk/gradle-plugins-target-platform/src/main/java/com/liferay/gradle/plugins/target/platform/extensions/TargetPlatformExtension.java

```diff
@@ -33,9 +36,36 @@
 public class TargetPlatformExtension {
 
 	public TargetPlatformExtension(Project project) {
+		_project = project;
 		_subprojects.addAll(project.getSubprojects());
 	}
 
+	public TargetPlatformExtension applyToConfiguration(
+		Iterable<?> applyToConfiguration) {
+
+		GUtil.addToCollection(_applyToConfiguration, applyToConfiguration);
+
+		Configuration targetPlatformBomsConfiguration =
+			GradleUtil.getConfiguration(
+				_project,
+				TargetPlatformPlugin.TARGET_PLATFORM_BOMS_CONFIGURATION_NAME);
+
+		TargetPlatformUtil.configureDependencyManagement(
+			_project, targetPlatformBomsConfiguration, _applyToConfiguration);
+
+		return this;
+	}
+
+	public TargetPlatformExtension applyToConfiguration(
+		Object... applyToConfiguration) {
+
+		return applyToConfiguration(Arrays.asList(applyToConfiguration));
+	}
+
+	public Set<?> getApplyToConfiguration() {
+		return _applyToConfiguration;
```

Comment:
Does it make sense to save the additional configurations? Will anything happen if people did `targetPlatform.applyToConfiguration.remove("foo")`?